app/controllers/external_contacts_controller.rb

```diff
@@ -7,17 +7,37 @@ def index
 
   def new
     authorize @project, :new_contact?
-    @contact = Contact::CreateProjectContactForm.new(contact: Contact::Project.new, project: @project)
+
+    @contact_form = if @project.is_a?(Conversion::Project)
+      NewConversionContactForm.new
+    else
+      NewTransferContactForm.new
+    end
   end
 
   def create
     authorize @project, :new_contact?
-    @contact = Contact::CreateProjectContactForm.new(contact: Contact::Project.new, project: @project, args: contact_params)
 
-    if @contact.save
-      redirect_to project_contacts_path(@project), notice: I18n.t("contact.create.success")
+    params = @project.is_a?(Conversion::Project) ? conversion_contact_params["contact_type"] : transfer_contact_params["contact_type"]
```

Comment:
Could we extract this to a method to better describe what it is doing, something like:

`def contact_type_from_params`